CreeDictionary/tests/API_tests/test_views.py

```diff
@@ -36,3 +36,32 @@ def test_click_in_text_no_params():
     response = c.get(reverse("cree-dictionary-word-click-in-text-api"))
 
     assert response.status_code == 400
+
+
+@pytest.mark.django_db
+def test_click_in_text_no_affix_search():
+    # This test guarantees affix search is turned off in click-in-text API
+
+    # Should affix search be turned on for click-in-tex, we would have a full page of results
+    # for the case of wapamew and a lot of results are from prefix/suffix search
+    #
+    # The user might not be certain what they are looking for on Itwewina, thus the need
+    # for relaxation and affix search. While click-in-text users searches for some exact cree they find on the web.
+    # So prefix/suffix search is unnecessary.
+    c = Client()
+
+    ascii_wapamew = "wapamew"
+    suffix_search_result = "asawâpamêw"
+
+    # fixme: I meant to do a comparison.
+    #  Normal searching with affix search should yields asawâpamêw
+    #  but not click in text search
+    #  however the next line (normal search) errors out and the traceback output is cryptic
+    #  I spent an hour debugging on this. It seems to error at template rendering. But accessing the same
+    #  url in the browser shows no error.
+    # normal_search_response = c.get(reverse("cree-dictionary-search") + f"?q={ascii_wapamew}").content.decode("utf-8")
```

Comment:
I'm honestly very concerned about this :/

Could you please get the normal search working in this test case? This issue sounds like the symptom of a much more concerning bug.

Reply:
will investigate more on this!